tests/LocalDateRangeTest.php

```diff
@@ -219,6 +219,42 @@ public function testToString()
         ));
     }
 
+    /**
+     * @dataProvider providerToDatePeriod
+     *
+     * @param string $dateTime The date-time string that will be parse()d by LocalDate.
+     * @param string $expected The expected output from the native DateTime object.
+     */
+    public function testToDatePeriod($a, $expectedStart, $expectedEnd)
+    {
+        $range = LocalDateRange::parse($a);
+
+        $period = $range->toDatePeriod();
+
+        $range_arr = iterator_to_array($range);
+        $period_arr = iterator_to_array($period);
+        $zip = array_map(null, $range_arr, $period_arr);
```

Comment:
Please don't use snake case for consistency with the rest of the project. `$rangeArray` and `$periodArray` would do fine.

Reply:
Done :)